Documentation/direct-tools.md

```diff
@@ -16,7 +16,7 @@ If your pod dies, your mount will die with it.
 After you have created a pool as described in the [Block Storage](ceph-block.md) topic, you can create a block image and mount it directly in a pod.
 This example will show how the Ceph rbd volume can be mounted in the toolbox pod.
 
-After you have started and connected to the [Rook toolbox](ceph-toolbox.md), proceed with the following commands in the toolbox.
+After you have started and connected to the [Rook toolbox](/cluster/examples/kubernetes/ceph/direct-mount.yaml), proceed with the following commands in the toolbox.
```

Comment:
Let's change this document to be self-contained. No need to link to other help topics. So please remove the link to the toolbox. This topic should not refer to the toolbox since we're going to call it the direct mount pod. For example something like this:

### Start the Direct Mount pod

To test mounting your Ceph volumes, start a pod with the necessary mounts. An example is provided in the examples test directory:
```
kubectl create -f cluster/examples/kubernetes/ceph/direct-mount.yaml
```
After the pod is started, connect to the pod
```
kubectl -n rook-ceph get pod -l app=rook-direct-mount
kubectl -n rook-ceph exec -it <pod> bash
```